### PR TITLE
Two variations of human scars

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

Weren't enough markings for humans and since there are tattoos; why not scars? However with the limited pixel range it was difficult to figure out anything so i just made two types; one left and one right (they look exactly the same). Right now they can be colored any color but ideally something that doesn't look horrendous lol anything further is out of the scope of this PR right now.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

## Breaking changes

**Changelog**

:cl: Ubaser
- add: Added two eye scar markings for humans and dwarves.

### CODE REVIEW

**Comment by EmoGarbage404** [comment]
i feel like you can just make the sprite a transparent black color and then make the marking skin-colored. That would basically just look like a scar without allowing it to be RGB hell.

**Comment by UbaserB** [comment]
> i feel like you can just make the sprite a transparent black color and then make the marking skin-colored. That would basically just look like a scar without allowing it to be RGB hell.

makes sense, i did try to do something similar but failed; ill try again.

**Comment by NeviNovat** [comment]
BUT I LIKE RGB HELL!

Honestly scars can look different depending on the damage and type of damage it was done to make them, some scar can remain deep red while some go pink and others white. Black may be used to indicate deep scars that leave a noticeable shadow.

**Comment by EmoGarbage404** [comment]
> Honestly scars can look different depending on the damage and type of damage it was done to make them, some scar can remain deep red while some go pink and others white. Black may be used to indicate deep scars that leave a noticeable shadow.

then write a skin tone validator. letting people make their scars bright neon pink is dumb and looks awful.

**Comment by NeviNovat** [comment]
> > Honestly scars can look different depending on the damage and type of damage it was done to make them, some scar can remain deep red while some go pink and others white. Black may be used to indicate deep scars that leave a noticeable shadow.
> 
> then write a skin tone validator. letting people make their scars bright neon pink is dumb and looks awful.

Fair point. That might be the best solution to! I wish I could code it, don't know where to start. One problem is  that while I know to code somewhat, there is so much code to go over just to understand one basic thing.

I like the RGB option still since I like more player freedoms. But I have no problems with restrictions, especially if proven that a freedom is "abused" continuously.

**Comment by UbaserB** [comment]
> i feel like you can just make the sprite a transparent black color and then make the marking skin-colored. That would basically just look like a scar without allowing it to be RGB hell.

i tried my best, the fallback is on pure white but it doesnt work. If i change the type or remove it, then the marking doesnt work at all. Not sure this is solvable without coding, sorry!

File: Resources/Locale/en-US/markings/scars.ftl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+marking-ScarEyeRight-eyescarright = Back Tattoo (Right)
+marking-ScarEyeRight = Eye Scar (Right)
+
+marking-ScarEyeLeft-eyescarleft = Back Tattoo (Left)
```
Comment (mirrorcult) [inline Resources/Locale/en-US/markings/scars.ftl:4]: whys it say back tattoo

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/markings/scars.ftl (hunk 1 above)
Comment (UbaserB) [reply Resources/Locale/en-US/markings/scars.ftl:4]: lol let me fix that

**Pull request merged**